app/lib/three_scale/api/responder.rb

```diff
@@ -75,10 +75,17 @@ def error_status
   def serializable
     @serializable ||= begin
       resource = options.fetch(:serialize){ self.resource }
-      resource = resource.is_a?(ActiveRecord::Relation) ? resource.to_a : resource
+      resource = resource.is_a?(ActiveRecord::Relation) ? ordered_relation(resource).to_a : resource
 
       resource
     end
   end
 
+  def ordered_relation(relation)
+    if System::Database.postgres? && relation.order_values.empty?
```

Comment:
We can later consider whether it is the right path making it the default behavior for all databases.

Reply:
Agree.